alerting/src/main/kotlin/org/opensearch/alerting/util/DocLevelMonitorQueries.kt

```diff
@@ -45,8 +51,11 @@ class DocLevelMonitorQueries(private val client: Client, private val clusterServ
 
     suspend fun initDocLevelQueryIndex(): Boolean {
         if (!docLevelQueryIndexExists()) {
-            val indexRequest = CreateIndexRequest(ScheduledJob.DOC_LEVEL_QUERIES_INDEX)
+            val alias = ScheduledJob.DOC_LEVEL_QUERIES_INDEX
+            val indexPattern = ScheduledJob.DOC_LEVEL_QUERIES_INDEX + INDEX_PATTERN_SUFIX
+            val indexRequest = CreateIndexRequest(indexPattern)
                 .mapping(docLevelQueriesMappings())
+                .alias(Alias(alias))
```

Comment:
Since before it was not an alias and it is now, have we tested BWC scenarios? If they came from the old Alerting plugin and created Document level monitors and then updated those monitors, would they behave correctly?

Reply:
Good catch! Creating alias would fail if index already exists with same name. We can append suffix to alias name, something like: "-alias" ?

Reply:
added checking if index exists and then deletion, as a first step before creating DOC_LEVEL_QUERIES_INDEX